command_line/sweep_to_stills.py

```diff
@@ -0,0 +1,230 @@
+    .type = float
+    .help = "Override for mosaic angle. If None, use the crystals mosaic angle, if"
+            "available"
+  half_mosaicity_deg = None
+    .type = float
+    .help = "Override for mosaic angle. If None, use the crystals mosaic angle, if"
+            "available"
+}
+"""
+)
+
+
+class Script(object):
+    """A class for running the script."""
+
+    def __init__(self):
+        """Initialise the script."""
+        from dials.util.options import OptionParser
+        import libtbx.load_env
+
+        # The script usage
+        usage = (
+            "usage: %s [options] [param.phil] filenames" % libtbx.env.dispatcher_name
+        )
+
+        # Create the parser
+        self.parser = OptionParser(
+            usage=usage,
+            phil=phil_scope,
+            read_experiments=True,
+            read_reflections=True,
+            check_format=False,
+            epilog=help_message,
+        )
+
+    def run(self):
+        """Execute the script."""
+        from dials.util.options import flatten_experiments, flatten_reflections
+        from dials.array_family import flex
+
+        # Parse the command line
+        params, options = self.parser.parse_args(show_diff_phil=True)
+
+        # Try to load the models and data
+        if not params.input.experiments:
+            print("No Experiments found in the input")
+            self.parser.print_help()
+            return
+        experiments = flatten_experiments(params.input.experiments)
+        reflections = flatten_reflections(params.input.reflections)
+        assert len(reflections) == 1
+        reflections = reflections[0]
+
+        new_experiments = ExperimentList()
+        new_reflections = flex.reflection_table()
+
+        # This is the subset needed to integrate
+        for key in [
+            "id",
+            "imageset_id",
+            "shoebox",
+            "bbox",
+            "intensity.sum.value",
+            "intensity.sum.variance",
+            "entering",
+            "flags",
+            "miller_index",
+            "panel",
+            "xyzobs.px.value",
+            "xyzobs.px.variance",
+        ]:
+            new_reflections[key] = type(reflections[key])()
+
+        for expt_id, experiment in enumerate(experiments):
+            # Get the goniometr setting matrix
+            goniometer_setting_matrix = sqr(
+                experiment.goniometer.get_setting_rotation()
+            )
+            goniometer_axis = col(experiment.goniometer.get_rotation_axis())
+            step = experiment.scan.get_oscillation()[1]
+
+            refls = reflections.select(reflections["id"] == expt_id)
+            _, _, _, _, z1, z2 = refls["bbox"].parts()
+
+            # Create an experiment for each scanpoint
+            for i_scanpoint in range(*experiment.scan.get_array_range()):
+                # The A matrix is the goniometer setting matrix for this scan point times the scan varying A matrix at this scan point
```

Comment:
Could you wrap the comments to max 88 characters line length to match the max line length of the code?